src/apdu_sign.c

```diff
@@ -182,9 +181,7 @@ static size_t baking_sign_complete(bool const send_hash, volatile uint32_t *flag
         case MAGIC_BYTE_ATTESTATION:
             guard_baking_authorized(&G.parsed_baking_data, &global.path_with_curve);
             result = perform_signature(true, send_hash);
-#ifdef HAVE_BAGL
-            ux_empty_screen();
-#endif
+            ui_initial_screen();
```

Comment:
This will recreate the entire home flow.
`update_baking_idle_screens` should be sufficient.

Reply:
```suggestion
            update_baking_idle_screen();
```